apps/playground/src/app/components/angular-tree-component/angular-tree-component/angular-tree-component.component.html

```
@@ -22,7 +25,10 @@ <h2>Basic</h2>
   </div>
 
   <h2>Read-only mode</h2>
-  <div class="sky-padding-even-default" id="screenshot-tree-view-basic">
+  <div
+    class="sky-theme-padding-inset-balanced-s"
+    id="screenshot-tree-view-basic"
```

HTMLHint (1.9.2)

[error] The id value [ screenshot-tree-view-basic ] must be unique.

(id-unique)